esp/esp/web/util/latex.py

```diff
@@ -140,59 +140,40 @@ def gen_latex(texcode, type='pdf', landscape=False, remove_files=False, stdout=P
     
     #   Set latex options
     latex_options = ['-interaction', 'nonstopmode', '-halt-on-error']
-    #   Set dvips options
-    dvips_options = ['-t', 'letter']
-    if landscape:
-        dvips_options = ['-t', 'letter,landscape']
```

Comment:
Yeah, it should -- it's set in the LaTeX file either in the `documentclass` options or with another package, and `pdflatex` will just get it from there.  I removed all the code that was used for that.